### DISCUSSION TITLE
why not increase filesLimit option in cgroup Resources struct?

### DISCUSSION THREAD

**Discussion by kamizjw** [opencontainers/runc#3595] [Q&A]

**Comment by kamizjw** [comment]
filesLimit is a commonly used option, I think it should be supported，cgroups is used for /proc/sys/fs/file-max

**Comment by kolyshkin** [comment]
What you propose doesn't make sense. cgroup.Resources is for resources that are controlled via cgroups.  /proc/sys/fs/file-max is a system-wide limit, not a per-container limit.

There is, although, Rlimits field in specs.Process which can be used to set RLIMIT_NOFILE for the process.
